test-plan.md

```diff
@@ -109,14 +111,26 @@ The Razor experience is available when you open a .cshtml file in a valid OmniSh
 ##### C# Diagnostics
 * Typing `@ThisDoesNotExist` results in an error being created and squiggled in the .cshtml file. NOTE: This error squiggly will be misaligned due to known issues.
 
+##### TagHelper completion
+Note that the pipe in the below examples indicates the cursor.
+* Typing `<environment>` prompts you with HTML completion for the `environment` tag and on completion commit auto-completes the closing tag.
+* Type `<cache |`, hit ctrl + space, search for `asp-vary-by-user`, commit the completion. Ensure that it auto-completes the attribute to: `<cache asp-vary-by-user`.
+* Type `<form |`, hit ctrl + space, search for `asp-route-...`, commit the completion. Ensure that it auto-completes the attribute to: `<form asp-route-|=""`. Type `foo` and then hit tab. Cursor should be inside of the double quotes: `<form asp-route-foo="|"`.
+
+##### Razor Diagnostics
+* Typing `@{ <strong> }` results in errors. One error should be about the "strong" tag missing a closing tag.
+* Fixing the error and typing `</strong>` i.e. `@{ <strong></strong> }` results in the diagnostic going away.
+
 ##### Known issues:
 - Error squiggles may be misaligned due to known issues.  
 
 #### Blazor
-The Blazor experience is available when you open a .cshtml file in a valid OmniSharp/Blazor project. To setup a test project to verify on you can do:  
+The Blazor experience is available when you open a .razor file in a valid OmniSharp/Blazor project. You should have a `dotnet` SDK >= 3.0.0-preview4 (`dotnet --version`), see https://github.com/dotnet/core-sdk for downloading newer SDKs.
+
+To setup a test project to verify on you can do:  
 1. `dotnet new -i Microsoft.AspNetCore.Blazor.Templates`
```

Comment:
Let's remove installing the client-side Blazor templates and just test with server-side Blazor. The tooling should be the same. Otherwise, we'll have to keep updating the version for the client-side Blazor templates.